### PR TITLE
README.md template: add OCA footer

### PR DESCRIPTION
**Pull request by sbidoul**
I found this short OCA description in several repos. It's good to have it in the template.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Yeah, I added it, but never move to template.

:+1:

**Comment by guewen** [comment]
:+1:

File: sample_files/README.md [hunk 1]
```diff
@@ -8,9 +8,17 @@
...
+This part will be replaced when running the oca-gen-addons-table script from OCA/maintainer-tools.
 [//]: # (end addons)
 
 Translation Status
 ------------------
 [![Transifex Status](https://www.transifex.com/projects/p/${ORG_NAME}-${REPO_NAME}-${BRANCH_NAME_WITH_DASH}/chart/image_png)](https://www.transifex.com/projects/p/${ORG_NAME}-${REPO_NAME}-${BRANCH_NAME_WITH_DASH})
+
+----
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
```
Comment (dreispt) [inline sample_files/README.md]: Use `[Odoo Community Association](http://odoo-community.org/)` and remove the link?

**Comment by lmignon** [comment]
:+1:

**Comment by sbidoul** [comment]
@pedrobaeza Note this footer is only present in ten repos or so. Maybe I'll do another mass commit to add it.

**Comment by moylop260** [comment]
:+1:

**Comment by elicoidal** [comment]
+1

**Comment by yvaucher** [comment]
This was merged a bit fast, @dreispt comment got no answer :sad:

**Comment by pedrobaeza** [comment]
Sorry, I wasn't saw it. Changed at https://github.com/OCA/maintainer-quality-tools/commit/c1778eafe9988770be32a65879199d0603d6b2f6

**Comment by yvaucher** [comment]
Thanks

**Pull request merged**